src/elements/content-sidebar/ActivitySidebarFilter.scss

```diff
@@ -0,0 +1,5 @@
+// Change size and weight of menu-toggle in ActivitySidebarFilter
```

Comment:
Can we remove this comment? It's describing the "what", which is visible in the code.